scipy/optimize/_bracket.py

```diff
@@ -205,7 +213,8 @@ def _bracket_root(func, xl0, xr0=None, *, xmin=None, xmax=None, factor=None,
     d[i] = x[i] - x0[i]
     d[ni] = limit[ni] - x[ni]
 
-    status = np.full_like(x, eim._EINPROGRESS, dtype=int)  # in progress
+
+    # We'll terminate with error status when initial bracket is invalid.
```

Comment:
```suggestion
```
This comment might belong somewhere else, but not here anymore.

Reply:
Ah, yeah I accidentally copy and pasted the part that went with the comment without moving the comment. I'm putting the status definition part back where you had it and setting `status[i] = eim._EINPUTERR` right after to minimize the diff. I think my motive was to put all the status stuff in one place, including finding the invalid brackets, but there's no need to do that.